### ISSUE TITLE
Basic loss functions

### ISSUE THREAD

**Issue by radeusgd** [Luna-Tensorflow/Luna-Tensorflow#64]
We want to have at least
- MSE
- MAE
- categorical_crossentropy,
- binary_crossentropy
- sparse_categorical_crossentropy
